sdk_commons/sdk_commons/helpers.py

```diff
@@ -0,0 +1,12 @@
+EVM_SIGNATURE_LEN = 130
+
+
+def break_evm_signature_into_components(signature: str) -> tuple[str, str, int]:
```

Comment:
I'm not sure about the name... something like get_evm_signature_components would not be better?

Reply:
done